src/formatDate.js

```diff
@@ -8,7 +8,48 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  let dateArr = [];
+  let resultString = '';
+  let index = 0;
+
+  if (date.includes('-')) {
+    dateArr = date.split('-');
+  } else if (date.includes('/')) {
+    dateArr = date.split('/');
+  } else {
+    dateArr = date.split('.');
+  }
+
+  for (let i = 0; i < toFormat.length - 1; i++) {
+    const fromIndex = fromFormat.indexOf(toFormat[i]);
+
+    index++;
+
+    if (toFormat[i] === 'YY' && fromFormat.includes('YYYY')) {
+      const number = dateArr[index - 1];
+
+      resultString += number.slice(2);
```

Comment:
When you're converting from 'YYYY' to 'YY', you're using the `slice` method to get the last two digits of the year. This is correct, but you should consider checking if the year is indeed 4 digits long to avoid unexpected results.